code/modules/shuttle/ripple.dm

```diff
@@ -8,6 +8,7 @@
 	anchored = TRUE
 	density = FALSE
 	layer = RIPPLE_LAYER
+	mouse_opacity = TRUE
```

Comment:
Oh, well, I didn't realise that temp/overlays were mouse_opacity = 0, and I want people to maybe pause to read my description before getting run over.